src/dotlottie-player.ts

```diff
@@ -343,219 +581,256 @@ export class DotLottiePlayer extends LitElement {
-      }
-      // Find desired animation and set the current animation index
-      const ret = this._manifest.animations.findIndex(element => element.id === animationId);
-      if (ret !== -1) {
-        this._active = ret;
-        if (this.src) {
-          if (!this._animations) throw `[dotLottie] No animations have been loaded.`;
-          this.load(this._animations[this._active]);
-        }
-      } else {
-        console.warn(`[dotLottie] No animation with the id '${animationId}' was found.`);
-      }
+      throw createError('Player failed to initialize.');
     }
   }
 
-  /**
-   * Get current animation's index
-   */
-  public getActive(): number {
-    return this._active;
-  }
-
   /**
    * Get current animation's id
    */
   public getActiveId(): string | null {
     if (!this._manifest.animations) return null;
-    return this._manifest.animations[this._active].id;
+
+    return this._manifest.animations[this._activeAnimationIndex].id;
   }
 
   /**
-   * Get current number of animations
+   * @returns The current number of animations
    */
   public animationCount(): number {
     if (this._animations) return this._animations.length;
+
     return 0;
   }
 
   /**
-   * Returns the lottie-web instance used in the component.
+   *
+   * @returns The current manifest.
+   */
+  public getManifest(): Manifest {
+    return this._manifest;
+  }
+
+  /**
+   * @returns The current lottie-web instance.
    */
   public getLottie(): AnimationItem {
     return this._lottie;
   }
 
+  /**
+   * @returns The current version of the dotLottie player and lottie-web.
+   */
   public getVersions(): Versions {
     return {
-      lottieWebVersion: LOTTIE_WEB_VERSION,
-      dotLottiePlayerVersion: DOTLOTTIE_PLAYER_VERSION,
+      lottieWebVersion: `${pkg.dependencies['lottie-web']}`,
+      dotLottiePlayerVersion: `${pkg.version}`,
     };
   }
 
+  private _requireAnimationsInTheManifest(): void {
+    if (!this._manifest.animations.length) {
+      throw createError(`No animations found in manifest.`);
+    }
+  }
+
+  private _requireAnimationsToBeLoaded(): void {
+    if (!this._animations?.length) {
+      throw createError(`No animations have been loaded.`);
+    }
+  }
+
+  /**
+   * Play the previous animation. The order is taken from the manifest.
+   */
+  public previous(playbackOptions?: PlaybackOptions): void {
+    this._requireAnimationsInTheManifest();
+    this._requireAnimationsToBeLoaded();
+
+    if (this._activeAnimationIndex === 0) {
+      this._activeAnimationIndex = this._manifest.animations.length - 1;
+    } else {
+      this._activeAnimationIndex -= 1;
+    }
+    this.play(this._activeAnimationIndex, playbackOptions);
+  }
+
+  /**
+   * Play the next animation. The order is taken from the manifest.
+   */
+  public next(playbackOptions?: PlaybackOptions): void {
+    this._requireAnimationsInTheManifest();
+    this._requireAnimationsToBeLoaded();
```

Comment:
I've noticed that this validation is performed in `next`, `prev`, `reset`. 

I wonder why don't we move it to the `play` method, as `next`, `prev`, `reset` calls `play` anyway

Reply:
Done ✅